deploy/examples/cluster-test.yaml

```diff
@@ -26,7 +26,10 @@ metadata:
   name: my-cluster
   namespace: rook-ceph # namespace:cluster
 spec:
-  dataDirHostPath: /var/lib/rook
```

Comment:
I'm hesitant to change this default for all test clusters. Even in minikube, persistence across reboot isn't usually needed since it's just for basic testing during development. How about just leaving the minikube recommendation as a comment, but keeping the default as `/var/lib/rook`?

Reply:
A comment is better than what we have now. I'll wait for more input on this.